src/App.jsx

```diff
@@ -69,13 +69,11 @@ function App() {
               }
             />
           </Route>
-
```

Comment:
Separating sections with a newline is a good practice, it distinguishes the different groups. Some coding styles are online to help understand like very popular AirBNB JS guidelines, or in the PHP land ESR guidelines.